### PR TITLE
Add loading component

### PR DESCRIPTION
**Pull request by EraAmate**

### CODE REVIEW

File: src/components/Loading.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  div {
+    display: inline-block;
+    position: absolute;
+    left: 8px;
+    width: 16px;
+    background: #fff;
+    animation: lds-facebook 1.2s cubic-bezier(0, 0.5, 0.5, 1) infinite;
+  }
+  div:nth-child(1) {
```
Comment (berlinum) [inline src/components/Loading.js]: I would change it to `div:nth-of-type(1)`

File: src/components/Loading.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    left: 8px;
+    animation-delay: -0.24s;
+  }
+  div:nth-child(2) {
```
Comment (berlinum) [inline src/components/Loading.js]: the same

File: src/components/Loading.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    left: 32px;
+    animation-delay: -0.12s;
+  }
+  div:nth-child(3) {
```
Comment (berlinum) [inline src/components/Loading.js]: and the same again ;)

File: src/components/Loading.js (hunk 1 above)
Comment (EraAmate) [reply src/components/Loading.js]: Well noted!! thanks!)

**Pull request merged**